src/components/comments/CommentCard.jsx

```diff
@@ -0,0 +1,61 @@
+import { useState } from "react";
+import styles from "./CommentCard.module.css";
+import CommentEditForm from "./CommentEditForm";
+import CommentView from "./CommentView";
+
+const CommentCard = ({ comment }) => {
+  const [isEditing, setIsEditing] = useState(false);
+  const [zIndex, setZIndex] = useState(0);
+  const handleEditClick = (e) => {
+    e.preventDefault();
+    setIsEditing(true);
+  };
+
+  const handleDeleteClick = (e) => {
+    e.preventDefault();
+  };
+
+  const handleSubmitEdit = (e) => {
+    e.preventDefault();
+    setIsEditing(false);
+  };
+
+  const handleCancelEdit = (e) => {
+    e.preventDefault();
+    setIsEditing(false);
+  };
+
+  const handleKebabOpen = () => {
+    setZIndex(1);
+  };
+
+  const handleKebabClose = () => {
+    setZIndex(0);
+  };
+
+  const containerStyle = {
+    zIndex: zIndex,
+  };
+
+  return (
+    <div style={containerStyle} className={styles["container"]}>
```

Comment:
위 코멘트처럼 컴포넌트 입장에서 중요한 데이터인 `isKebabOpen`에 따른 스타일링 분기를 만들어주면 이렇게 되겠죠?

```suggestion
    <div className={`${styles.container} ${isKebabOpen ? styles.open : ""}`}>
```